external/audiocodec/streaming/player.c

```diff
@@ -662,11 +600,12 @@ int _init_decoder(pv_player_p player)
 
 		player->config_func(player->cb_data, player->audio_type, player->dec_ext);
 
+		PVMP4AudioDecoderResetBuffer(player->dec_mem);
```

Comment:
I have a question, AAC buffer is cleared here, but we don't have to clean mp3 buffer in above?